recipe/meta.yaml

```diff
@@ -11,115 +11,126 @@ source:
   sha256: 9db4db2133a52e48af043fbb3bb920a77886f64a6a1c8cd56adee41828487fc6
 
 build:
-  number: 1
+  number: 0
+  skip: true  # [py<38]
 
 outputs:
   - name: allure-python-commons
```

Comment:
```suggestion
  - name: allure-python-commons
  - script:
       - install.sh   # [unix]
       - install.bat  # [win]
```

Reply:
Something like this? @skupr-anaconda

Reply:
Yep

Reply:
thank you @Jrice1317 and @skupr-anaconda